src/main/resources/staff/templates/document/defendantPinLetter.html

```diff
@@ -51,7 +51,7 @@
         </td>
         <td class="no-spacing">
           <p class="address" style="float:right">
-            County Court Business Centre Online Court<br/>
+            County Court Business Centre Online Civil Money Claims<br/>
```

Comment:
Sometimes we have these in two lines, sometimes in one line. Is there any rule? I guess we should aim to make it consistent and have two lines.

Reply:
After demo to PO, decided to break in two line where we have image logo, otherwise it should be one line as per PO (Marc)

Reply:
Discussed with PO and he agrees to make these consistent on two lines.